src/ethereum_test_fixtures/blockchain.py

```diff
@@ -483,6 +483,8 @@ class FixtureCommon(BaseFixture):
     """
 
     fork: str = Field(..., alias="network")
+    chain_id: ZeroPaddedHexNumber = Field(1, alias="chainId")  # type: ignore
```

Comment:
I think it should not be necessary to add the alias here.

Reply:
Nice catch!